netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -59,11 +65,34 @@ public TkStart(final Base bse) {
 
     @Override
     public Response act(final Request req) throws IOException {
-        final long number = new RqAlias(this.base, req).alias().inbox().start();
+        final Inbox inbox = new RqAlias(this.base, req).alias().inbox();
+        final long number = inbox.start();
+        final Bout bout = inbox.bout(number);
+        final StringBuilder msg = new StringBuilder(
+            String.format("new bout #%d started", number)
+        );
+        final Href href = new RqHref.Base(req).href();
+        final Iterator<String> invite = href.param("invite").iterator();
+        while (invite.hasNext()) {
+            final String friend = invite.next();
+            try {
+                bout.friends().invite(friend);
+            } catch (final Friends.UnknownAliasException ex) {
+                throw new RsFailure(ex);
```

Comment:
at this point the bout was already created, right? Shouldn't  we do something about it?